frontend/src/main/components/Chat/ChatToggle.js

```diff
@@ -11,16 +12,16 @@ const chatButtonStyle = {
     right: '20px',
   };
 
-
-const ChatToggle = ({ toggleChatWindow, isChatOpen }) => 
+const ChatToggle = ({ toggleChatWindow}) => 
 {
+    
     return (
         <Button
-            style={chatButtonStyle}
+            style={chatContainerStyle}
             onClick={toggleChatWindow}
             data-testid="ChatToggle"
         >
-            {!isChatOpen? '▲' : '▼'}
+            <img src={ChatSymbol} style = {chatContainerStyle} alt = "Chat Symbol"/>
```

Comment:
Change svg based on whether chatroom is visible to the end user.

Reply:
Thanks for the suggestion! That's a really nice website, makes life easier.